### ISSUE TITLE
Add 'on hold' ticket state?

### ISSUE THREAD

**Issue by flashadvocate** [ClanAODDev/tracker_v3#246]
Sometimes a ticket comes in that can't be handled immediately, but isn't waiting on something from the admin. For these cases, it might make sense to add an "on hold" state to represent that some further action needs to take place before a ticket can be completed.

**Comment by bashtech** [comment]
I was looking at this a bit last week and have a draft almost ready to go. What do you think about adding two new ticket states: `on_hold` and `in_progress`?

In Progress is optional but could be used when a ticket requires more time to be completed, such as media team requests, while still providing an indication to the caller that the process is moving. Tickets could still go directly from New or Assigned to Resolved. It also is the state that a ticket could be placed in when it's moved out of the On Hold state if it was not immediately resolved.

![image](https://github.com/ClanAODDev/tracker_v3/assets/112572/f8a04130-7f6f-4081-9921-0cfb8785f59e)

**Comment by flashadvocate** [comment]
I had always imagined the "assigned" status filling the niche of the in progress state. Granted some of the auto-assigned tickets don't have any real feedback from the new owner that work is being done. I'm not opposed to this state, but it would mostly rely on the MSGT+ group to purposefully hit the "in progress" button vs just doing the work and then resolving. Might be worth asking about

**Comment by flashadvocate** [comment]
Hold tight on this for the time being. The MSGT+ group doesn't seem too thrilled about additional states and thinks that the existing ones serve the purpose pretty well
